### PR TITLE
Add Aaron Jorbin

### PR DESCRIPTION
**Pull request by aaronjorbin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ Let's invest in the folks who create the software that empowers us to create a l
 
 ## Contributors
 
+[Aaron Jorbin](https://aaron.jorb.in)
+  - Twitter: [@allie_nimmons](https://twitter.com/aaronjorbin)
```
Comment (jeffpaul) [inline README.md]: should be `@aaronjorbin`

File: README.md [hunk 2]
```diff
@@ -4,6 +4,11 @@ Let's invest in the folks who create the software that empowers us to create a l
...
+  - WordPress: [@allienimmons](https://profiles.wordpress.org/jorbin)
```
Comment (jeffpaul) [inline README.md]: should be `@jorbin`

File: README.md (hunk 1 above)
Comment (aurooba) [reply README.md]: Thanks for reviewing these, @jeffpaul!

File: README.md (hunk 2 above)
Comment (aurooba) [reply README.md]: Thanks for reviewing these, @jeffpaul!

**Pull request merged**